.github/workflows/build.yml

```diff
@@ -91,7 +92,8 @@ jobs:
             openrc \
             openssl-dev \
             pkgconfig \
-            rpcsvc-proto-dev
+            rpcsvc-proto-dev \
+            wolfssl-dev
```

Comment:
Isn't it misleading to install the wolfssl-dev for this OS if it won't be usable by netatalk?

Reply:
(Same comment for other OSes except Debian and Ubuntu below)

Reply:
I thought it was a good way to test the logic of the macro. Will remove now I know it works